vc-trusted-issuer-data-service/vc_til_service.py

```diff
@@ -0,0 +1,305 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2021 Future Internet Consulting and Development Solutions S.L.
+
+# This file is part of BAE NGSI Dataset plugin.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+import os
+import re
+import jwt
+import json
+import requests
+from requests.exceptions import HTTPError
+import uuid
+
+import time
+from datetime import datetime, timedelta
+
+from urllib.parse import urlparse
+from urllib.parse import parse_qs
+
+from base64 import urlsafe_b64encode
+
+from django.conf import settings
+
+from wstore.asset_manager.resource_plugins.plugin import Plugin
```

Comment:
how does the package 'wstore' being imported? My IDE can't find this dependency

Reply:
The plugin is executed by the charging backend. There this package is defined.